doc/sphinxman/source/scf.rst

```diff
@@ -745,6 +751,33 @@ To avoid this, either set |scf__df_basis_scf| to an auxiliary
 basis set defined for all atoms in the system, or set |scf__df_scf_guess|
 to false, which disables this acceleration entirely.
 
+COSX Exchange
+~~~~~~~~~~~~~
+
+The semi-numerical COSX algorithm (described in [Neese:2009:98]_) is so-called
+because the two-electron ERIs are evaluated analytically over one electron 
+coordinate and numerically over the other electron coordinate. The numerical 
```

Comment:
```suggestion
The semi-numerical COSX algorithm (described in [Neese:2009:98]_) evaluates two-electron ERIs
analytically over one electron coordinate and numerically over the other electron coordinate. The numerical
```

Reply:
One could also mention the connection to the pseudospectral method by Friesner here?

Reply:
Added a sentence about Friesner